doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -108,10 +137,10 @@ Show Contents of a Dataverse
 .. code-block:: bash
 
   export API_TOKEN=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-  export ALIAS=root
   export SERVER_URL=https://demo.dataverse.org
-  
-  curl -H X-Dataverse-key:$API_TOKEN $SERVER_URL/api/dataverses/$ALIAS/contents
+  export alias=root
```

Comment:
@pkiraly was there a reason to switch the variable formatting from uppercase to lowercase? I believe it would be more consistent with the other variables like $API_TOKEN, and would stand out more in the curl commands, if all lowercase variables were uppercase.

Reply:
There was inconsistency in the variable case, e.g.:
```
curl -H X-Dataverse-key:$API_TOKEN $SERVER_URL/api/dataverses/$id
```
I left SERVER_URL and API_TOKEN uppercate to suggest they are constant within a script or API session, and made other lowercase if they were not already in lowercase.

Do you think all variables should be uppercase? It might affect other API pages as well.

Reply:
Hi @pkiraly it is really just a matter of style and while there is some utility in differentiating between configuration variable vs. data variables, I think it'd be better for them all to standout - it makes it significantly more obvious that those are placeholders that need to be changed.

(I'm willing to reconsider if there is a known convention for having these be different, but I couldn't find any in my quick attempt to review online)

As you said, it will likely affect other API pages, and if you have time to make updates, that would be appreciated. If not, then we can just tackle incrementally as we make edits to those pages.

Reply:
@mheppler, @scolapasta I changed them uppercase. I found one thing which also needs some conversation: the identifier of a dataverse should be ALIAS or ID? Mostly it is ID, but there are some ALIAS-es here and there.

Reply:
We used to call it alias (and still do in db and code) but it later switched to thinking of it as identifier (as that was more clear to outsiders). So probably best to go with ID.

Reply:
@scolapasta I replaced ALIAS with ID in the Dataverse related native API calls.